[2019-01-12 16:43:33] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 16:43:33] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 16:43:33] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@116b0b32, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@4f22fd5d, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@26d445e9, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@6e7f29d5, org.springframework.test.context.support.DirtiesContextTestExecutionListener@27048db3, org.springframework.test.context.transaction.TransactionalTestExecutionListener@7f94541b, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@7bebad0d, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@6c4d0224, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6f27730b, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@32cec678, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@14eb96e9, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@10b3ea72]
[2019-01-12 16:43:36] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$e34fe85f] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 16:43:36] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$9a65337] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 16:43:42] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-8926014893572678926/Application.yml
[2019-01-12 16:43:42] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-8926014893572678926/application.yml
[2019-01-12 16:43:42] [main] INFO  o.s.c.b.c.PropertySourceBootstrapConfiguration - Located property source: CompositePropertySource {name='configService', propertySources=[MapPropertySource {name='https://github.com/Romeh/config.git/Application.yml'}, MapPropertySource {name='https://github.com/Romeh/config.git/application.yml'}]}
[2019-01-12 16:43:42] [main] INFO  o.m.s.d.integration.GetVersionStep - The following profiles are active: INTEGRATION_TEST
[2019-01-12 16:43:45] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
[2019-01-12 16:43:45] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 142ms. Found 1 repository interfaces.
[2019-01-12 16:43:46] [main] INFO  o.s.cloud.context.scope.GenericScope - BeanFactory id=3ca24905-7235-3b69-89c9-d0a8eb7504f1
[2019-01-12 16:43:46] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$e34fe85f] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 16:43:46] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$ed8c503a] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 16:43:46] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$9a65337] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 16:43:47] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 0 (http)
[2019-01-12 16:43:47] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 16:43:47] [main] INFO  o.a.catalina.core.StandardService - Starting service [Tomcat]
[2019-01-12 16:43:47] [main] INFO  o.a.catalina.core.StandardEngine - Starting Servlet Engine: Apache Tomcat/9.0.12
[2019-01-12 16:43:47] [main] INFO  o.a.c.core.AprLifecycleListener - The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [C:\Program Files\Java\jdk1.8.0_77\jre\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files (x86)\HP SimplePass 2011\x64;C:\Program Files (x86)\HP SimplePass 2011\;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Shared\12.0\DLLShared\;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\Internet Explorer;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\Git\cmd;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLI\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Python3.6\Scripts\;C:\Python3.6\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\MongoDB\Server\3.0\bin;C:\apache-maven-3.2.2\bin;C:\Python27_;C:\Python27_\Scripts;C:\Program Files\Git\bin;C:\Program Files\Java\jdk1.8.0_77\bin;C:\Users\Moorthy\AppData\Local\Microsoft\WindowsApps;C:\workspace\gradle-4.1\bin;c:\spark\bin;C:\Program Files (x86)\GnuWin32\bin;C:\software install\apache-ant-1.10.5\bin;C:\software install\apache-ant-1.10.5\lib;C:\Program Files (x86)\7-Zip;C:\Users\Moorthy\AppData\Roaming\npm;.]
[2019-01-12 16:43:48] [main] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2019-01-12 16:43:48] [main] INFO  o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 5219 ms
[2019-01-12 16:43:49] [main] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Starting embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false', username='sa'
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'characterEncodingFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'webMvcMetricsFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'formContentFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'requestContextFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'httpTraceFilter' to: [/*]
[2019-01-12 16:43:50] [main] INFO  o.s.b.w.s.ServletRegistrationBean - Servlet dispatcherServlet mapped to [/]
[2019-01-12 16:43:51] [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
[2019-01-12 16:43:51] [main] INFO  org.hibernate.Version - HHH000412: Hibernate Core {5.3.7.Final}
[2019-01-12 16:43:51] [main] INFO  org.hibernate.cfg.Environment - HHH000206: hibernate.properties not found
[2019-01-12 16:43:51] [main] INFO  o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
[2019-01-12 16:43:52] [main] INFO  org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
[2019-01-12 16:43:53] [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 16:43:55] [main] WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2019-01-12 16:43:56] [main] INFO  o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
[2019-01-12 16:43:56] [main] INFO  s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[2019-01-12 16:43:57] [main] INFO  o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
[2019-01-12 16:43:58] [main] INFO  o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService 'taskScheduler'
[2019-01-12 16:43:58] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
[2019-01-12 16:43:58] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
[2019-01-12 16:43:59] [main] INFO  s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
[2019-01-12 16:43:59] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_1
[2019-01-12 16:43:59] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_2
[2019-01-12 16:43:59] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_3
[2019-01-12 16:43:59] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 16:43:59] [main] INFO  o.a.tomcat.util.net.NioSelectorPool - Using a shared selector for servlet write/read
[2019-01-12 16:43:59] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 52037 (http) with context path ''
[2019-01-12 16:43:59] [main] INFO  o.m.s.d.integration.GetVersionStep - Started GetVersionStep in 25.988 seconds (JVM running for 29.702)
[2019-01-12 16:44:00] [http-nio-auto-1-exec-1] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
[2019-01-12 16:44:00] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
[2019-01-12 16:44:00] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Completed initialization in 54 ms
[2019-01-12 16:44:01] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 16:44:01] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 16:44:01] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@7b80655e, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@3546b740, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@2b583ea5, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@48301adb, org.springframework.test.context.support.DirtiesContextTestExecutionListener@30623c16, org.springframework.test.context.transaction.TransactionalTestExecutionListener@301e71ed, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@5ad9dc5d, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@1277e31f, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@33d4ffcb, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@70066145, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@6429702c, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@496b18ad]
[2019-01-12 16:44:01] [Thread-17] INFO  o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService 'taskScheduler'
[2019-01-12 16:44:01] [Thread-17] INFO  o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
[2019-01-12 16:44:01] [Thread-17] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 16:44:01] [Thread-17] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Shutting down embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false'
[2019-01-12 16:44:01] [Thread-3] WARN  o.s.c.c.s.e.MultipleJGitEnvironmentRepository - Failed to delete temporary directory on exit: java.nio.file.AccessDeniedException: C:\Users\Moorthy\AppData\Local\Temp\config-repo-8926014893572678926\.git\objects\pack\pack-ac4cc2b653f3e9d9e60018dd8fbaa6fa735e7123.idx
[2019-01-12 17:07:11] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:07:11] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:07:11] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@2f3a0f37, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@7ec8db0c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@4f1a5b93, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@22ee1ad7, org.springframework.test.context.support.DirtiesContextTestExecutionListener@5dd2ea0a, org.springframework.test.context.transaction.TransactionalTestExecutionListener@65fa4a07, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@782143e8, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@2a4bd699, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6003eb60, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@571e2f15, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@2552181d, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@51d9fd30]
[2019-01-12 17:07:13] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:07:14] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:07:20] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-1673436236833824755/Application.yml
[2019-01-12 17:07:20] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-1673436236833824755/application.yml
[2019-01-12 17:07:20] [main] INFO  o.s.c.b.c.PropertySourceBootstrapConfiguration - Located property source: CompositePropertySource {name='configService', propertySources=[MapPropertySource {name='https://github.com/Romeh/config.git/Application.yml'}, MapPropertySource {name='https://github.com/Romeh/config.git/application.yml'}]}
[2019-01-12 17:07:20] [main] INFO  o.m.s.d.integration.GetVersionStep - The following profiles are active: INTEGRATION_TEST
[2019-01-12 17:07:22] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
[2019-01-12 17:07:23] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 189ms. Found 1 repository interfaces.
[2019-01-12 17:07:23] [main] INFO  o.s.cloud.context.scope.GenericScope - BeanFactory id=cbfe51bb-3762-39d5-af4d-713a4c28d593
[2019-01-12 17:07:24] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:07:24] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$92511aca] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:07:24] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:07:25] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 0 (http)
[2019-01-12 17:07:25] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:07:25] [main] INFO  o.a.catalina.core.StandardService - Starting service [Tomcat]
[2019-01-12 17:07:25] [main] INFO  o.a.catalina.core.StandardEngine - Starting Servlet Engine: Apache Tomcat/9.0.12
[2019-01-12 17:07:25] [main] INFO  o.a.c.core.AprLifecycleListener - The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [C:\Program Files\Java\jdk1.8.0_77\jre\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files (x86)\HP SimplePass 2011\x64;C:\Program Files (x86)\HP SimplePass 2011\;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Shared\12.0\DLLShared\;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\Internet Explorer;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\Git\cmd;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLI\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Python3.6\Scripts\;C:\Python3.6\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\MongoDB\Server\3.0\bin;C:\apache-maven-3.2.2\bin;C:\Python27_;C:\Python27_\Scripts;C:\Program Files\Git\bin;C:\Program Files\Java\jdk1.8.0_77\bin;C:\Users\Moorthy\AppData\Local\Microsoft\WindowsApps;C:\workspace\gradle-4.1\bin;c:\spark\bin;C:\Program Files (x86)\GnuWin32\bin;C:\software install\apache-ant-1.10.5\bin;C:\software install\apache-ant-1.10.5\lib;C:\Program Files (x86)\7-Zip;C:\Users\Moorthy\AppData\Roaming\npm;.]
[2019-01-12 17:07:25] [main] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2019-01-12 17:07:25] [main] INFO  o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 4803 ms
[2019-01-12 17:07:26] [main] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Starting embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false', username='sa'
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'characterEncodingFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'webMvcMetricsFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'formContentFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'requestContextFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'httpTraceFilter' to: [/*]
[2019-01-12 17:07:27] [main] INFO  o.s.b.w.s.ServletRegistrationBean - Servlet dispatcherServlet mapped to [/]
[2019-01-12 17:07:28] [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
[2019-01-12 17:07:28] [main] INFO  org.hibernate.Version - HHH000412: Hibernate Core {5.3.7.Final}
[2019-01-12 17:07:28] [main] INFO  org.hibernate.cfg.Environment - HHH000206: hibernate.properties not found
[2019-01-12 17:07:29] [main] INFO  o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
[2019-01-12 17:07:29] [main] INFO  org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
[2019-01-12 17:07:31] [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:07:32] [main] WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2019-01-12 17:07:33] [main] INFO  o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
[2019-01-12 17:07:34] [main] INFO  s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[2019-01-12 17:07:34] [main] INFO  o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:07:35] [main] INFO  o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService 'taskScheduler'
[2019-01-12 17:07:36] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
[2019-01-12 17:07:36] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
[2019-01-12 17:07:37] [main] INFO  s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
[2019-01-12 17:07:37] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_1
[2019-01-12 17:07:37] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_2
[2019-01-12 17:07:37] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_3
[2019-01-12 17:07:37] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:07:37] [main] INFO  o.a.tomcat.util.net.NioSelectorPool - Using a shared selector for servlet write/read
[2019-01-12 17:07:37] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 52364 (http) with context path ''
[2019-01-12 17:07:37] [main] INFO  o.m.s.d.integration.GetVersionStep - Started GetVersionStep in 26.083 seconds (JVM running for 29.473)
[2019-01-12 17:07:38] [http-nio-auto-1-exec-1] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
[2019-01-12 17:07:38] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
[2019-01-12 17:07:38] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Completed initialization in 58 ms
[2019-01-12 17:07:39] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:07:39] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:07:39] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@2b2ffb04, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@4379ac7a, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@49afe410, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@4b923c2c, org.springframework.test.context.support.DirtiesContextTestExecutionListener@1f311981, org.springframework.test.context.transaction.TransactionalTestExecutionListener@11a57c61, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@271c8521, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@7db5beb2, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6adf5d94, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@502372da, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@1355c072, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@43c5ac84]
[2019-01-12 17:07:39] [Thread-17] INFO  o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService 'taskScheduler'
[2019-01-12 17:07:39] [Thread-17] INFO  o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:07:39] [Thread-17] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:07:39] [Thread-17] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Shutting down embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false'
[2019-01-12 17:07:39] [Thread-3] WARN  o.s.c.c.s.e.MultipleJGitEnvironmentRepository - Failed to delete temporary directory on exit: java.nio.file.AccessDeniedException: C:\Users\Moorthy\AppData\Local\Temp\config-repo-1673436236833824755\.git\objects\pack\pack-ac4cc2b653f3e9d9e60018dd8fbaa6fa735e7123.idx
[2019-01-12 17:20:02] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:20:03] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:20:03] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@2f3a0f37, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@7ec8db0c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@4f1a5b93, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@22ee1ad7, org.springframework.test.context.support.DirtiesContextTestExecutionListener@5dd2ea0a, org.springframework.test.context.transaction.TransactionalTestExecutionListener@65fa4a07, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@782143e8, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@2a4bd699, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6003eb60, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@571e2f15, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@2552181d, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@51d9fd30]
[2019-01-12 17:20:06] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:20:06] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:20:12] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-7165538557564826831/Application.yml
[2019-01-12 17:20:12] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-7165538557564826831/application.yml
[2019-01-12 17:20:12] [main] INFO  o.s.c.b.c.PropertySourceBootstrapConfiguration - Located property source: CompositePropertySource {name='configService', propertySources=[MapPropertySource {name='https://github.com/Romeh/config.git/Application.yml'}, MapPropertySource {name='https://github.com/Romeh/config.git/application.yml'}]}
[2019-01-12 17:20:13] [main] INFO  o.m.s.d.integration.GetVersionStep - The following profiles are active: INTEGRATION_TEST
[2019-01-12 17:20:15] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
[2019-01-12 17:20:15] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 154ms. Found 1 repository interfaces.
[2019-01-12 17:20:16] [main] INFO  o.s.cloud.context.scope.GenericScope - BeanFactory id=26ae97b7-cd7d-38cc-b55f-c490d254bb23
[2019-01-12 17:20:16] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:20:16] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$92511aca] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:20:16] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:20:18] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 0 (http)
[2019-01-12 17:20:18] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:20:18] [main] INFO  o.a.catalina.core.StandardService - Starting service [Tomcat]
[2019-01-12 17:20:18] [main] INFO  o.a.catalina.core.StandardEngine - Starting Servlet Engine: Apache Tomcat/9.0.12
[2019-01-12 17:20:18] [main] INFO  o.a.c.core.AprLifecycleListener - The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [C:\Program Files\Java\jdk1.8.0_77\jre\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files (x86)\HP SimplePass 2011\x64;C:\Program Files (x86)\HP SimplePass 2011\;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Shared\12.0\DLLShared\;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\Internet Explorer;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\Git\cmd;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLI\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Python3.6\Scripts\;C:\Python3.6\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\MongoDB\Server\3.0\bin;C:\apache-maven-3.2.2\bin;C:\Python27_;C:\Python27_\Scripts;C:\Program Files\Git\bin;C:\Program Files\Java\jdk1.8.0_77\bin;C:\Users\Moorthy\AppData\Local\Microsoft\WindowsApps;C:\workspace\gradle-4.1\bin;c:\spark\bin;C:\Program Files (x86)\GnuWin32\bin;C:\software install\apache-ant-1.10.5\bin;C:\software install\apache-ant-1.10.5\lib;C:\Program Files (x86)\7-Zip;C:\Users\Moorthy\AppData\Roaming\npm;.]
[2019-01-12 17:20:18] [main] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2019-01-12 17:20:18] [main] INFO  o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 5895 ms
[2019-01-12 17:20:19] [main] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Starting embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false', username='sa'
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'characterEncodingFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'webMvcMetricsFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'formContentFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'requestContextFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'httpTraceFilter' to: [/*]
[2019-01-12 17:20:22] [main] INFO  o.s.b.w.s.ServletRegistrationBean - Servlet dispatcherServlet mapped to [/]
[2019-01-12 17:20:25] [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
[2019-01-12 17:20:26] [main] INFO  org.hibernate.Version - HHH000412: Hibernate Core {5.3.7.Final}
[2019-01-12 17:20:26] [main] INFO  org.hibernate.cfg.Environment - HHH000206: hibernate.properties not found
[2019-01-12 17:20:28] [main] INFO  o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
[2019-01-12 17:20:31] [main] INFO  org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
[2019-01-12 17:20:37] [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:20:41] [main] WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2019-01-12 17:20:41] [main] INFO  o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
[2019-01-12 17:20:47] [qtp1851898486-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:47] [qtp1851898486-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:47] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[1]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:47] [qtp1851898486-39] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:47] [qtp1851898486-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=7520fd85a825b32d8d30b2b6763c7be2e61def0cbab7467a5e0d9e5a7c6e8102
header name: X-Amz-Date : value: 20190112T222046Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 0/0/500
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:47] [qtp1851898486-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: bb6a6152-e6f6-4270-941d-dcb55ae5e969
header name: Date : value: Sat, 12 Jan 2019 22:20:47 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:47] [qtp1851898486-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:47] [qtp1851898486-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:47] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[2]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:47] [qtp1851898486-33] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:132)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:47] [qtp1851898486-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=2461b42d4401f0f89b0e3d3aee916d7e2966cdb307a17b34cca44c91df149da3
header name: X-Amz-Date : value: 20190112T222047Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 1/0/495
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:47] [qtp1851898486-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: ec088d77-e4f2-4525-b810-e99e26becf7f
header name: Date : value: Sat, 12 Jan 2019 22:20:47 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:47] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:47] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:47] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[3]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:47] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:47] [qtp1851898486-38] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:47] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=34aa4ac6923c291e3d28ec96a5d2b903bfaa40ea78c45c956ea224dca161fbf2
header name: X-Amz-Date : value: 20190112T222047Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 2/0/490
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:47] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: e36770fc-d492-4a01-a08c-8b78c67fa822
header name: Date : value: Sat, 12 Jan 2019 22:20:47 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:47] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:47] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:48] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[4]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:48] [qtp1851898486-37] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=0d501dfe0e815ad7c5e95b76d783c8d8a9fbd3660de182b5ffa15967a0786eeb
header name: X-Amz-Date : value: 20190112T222047Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 3/2/485
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 730b3ebd-e6e3-4464-a6f4-929eb32c5207
header name: Date : value: Sat, 12 Jan 2019 22:20:47 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:48] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[5]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:48] [qtp1851898486-38] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=94624fd723216f2857922c60779675eb99410ac00db25cf65d72d0a9ad9d722a
header name: X-Amz-Date : value: 20190112T222048Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 4/178/480
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 0eb6479c-4554-44e1-9363-ff264f5060a8
header name: Date : value: Sat, 12 Jan 2019 22:20:48 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:48] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[6]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:48] [qtp1851898486-37] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=80d4ec2e6baa879af0706f41b2ae2be60cadadf5bbcba1f2c8530ae4dfe52c89
header name: X-Amz-Date : value: 20190112T222048Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 5/48/475
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:48] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 3b7acd0e-74e2-4e6f-a9fb-52280e39716c
header name: Date : value: Sat, 12 Jan 2019 22:20:48 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:48] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[7]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:48] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:48] [qtp1851898486-38] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=f54bb0873b3d16447e08d69ddb68c67f91bc991211a6db4e8b4d1f9bd0c6ad10
header name: X-Amz-Date : value: 20190112T222048Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 6/409/470
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:48] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 9c7802f5-4178-4a50-ad29-e9f06a3870b3
header name: Date : value: Sat, 12 Jan 2019 22:20:48 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:50] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:50] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:50] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[8]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:50] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:50] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:50] [qtp1851898486-37] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:50] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=8236ab46cf10891aa8162ca10c838a72157294e8d73cb49f93396f4209a2b2a4
header name: X-Amz-Date : value: 20190112T222050Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 7/1244/465
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:50] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: e8bfca96-8192-456f-97e3-65b01c102380
header name: Date : value: Sat, 12 Jan 2019 22:20:50 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:52] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:52] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:52] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[9]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:52] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:52] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:52] [qtp1851898486-38] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:52] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=101c37c93ead892dfb7e1fce4905578a82de3d25e8bf5c3798c5075221e8f7b4
header name: X-Amz-Date : value: 20190112T222052Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 8/2215/460
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:52] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 18579683-ba5d-4ba5-a645-f62cece6ffa6
header name: Date : value: Sat, 12 Jan 2019 22:20:52 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:20:54] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:20:54] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:20:54] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[10]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:20:54] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:20:54] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:20:54] [qtp1851898486-37] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:20:54] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=586905a12c11acb95ad10be4217abc076f308787bbd61b1455699858a87891b6
header name: X-Amz-Date : value: 20190112T222054Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 9/2350/455
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:20:54] [qtp1851898486-37] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: fbacc6d7-b41a-4bfd-9a46-e81e83df6b32
header name: Date : value: Sat, 12 Jan 2019 22:20:54 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:21:02] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:21:02] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:21:02] [SQLiteQueue[]] WARN  com.almworks.sqlite4java - [sqlite] cannot open DB[11]: com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
[2019-01-12 17:21:02] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: error running job queue
com.almworks.sqlite4java.SQLiteException: [-91] cannot load library: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:97)
	at com.almworks.sqlite4java.SQLiteConnection.open0(SQLiteConnection.java:1441)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:282)
	at com.almworks.sqlite4java.SQLiteConnection.open(SQLiteConnection.java:293)
	at com.almworks.sqlite4java.SQLiteQueue.openConnection(SQLiteQueue.java:464)
	at com.almworks.sqlite4java.SQLiteQueue.queueFunction(SQLiteQueue.java:641)
	at com.almworks.sqlite4java.SQLiteQueue.runQueue(SQLiteQueue.java:623)
	at com.almworks.sqlite4java.SQLiteQueue.access$000(SQLiteQueue.java:77)
	at com.almworks.sqlite4java.SQLiteQueue$1.run(SQLiteQueue.java:205)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: no sqlite4java-win32-x64-1.0.392 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.almworks.sqlite4java.Internal.tryLoadFromSystemPath(Internal.java:352)
	at com.almworks.sqlite4java.Internal.loadLibraryX(Internal.java:124)
	at com.almworks.sqlite4java.SQLite.loadLibrary(SQLite.java:95)
	... 9 common frames omitted
[2019-01-12 17:21:02] [SQLiteQueue[]] ERROR com.almworks.sqlite4java - [sqlite] SQLiteQueue[]: stopped abnormally, reincarnation is not possible for in-memory database
[2019-01-12 17:21:02] [qtp1851898486-38] ERROR c.a.s.d.l.s.LocalDynamoDBServerHandler - Unexpected exception occured
java.lang.NullPointerException: null
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.initializeMetadataTables(SQLiteDBAccess.java:391)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:227)
	at com.amazonaws.services.dynamodbv2.local.shared.access.sqlite.SQLiteDBAccess.<init>(SQLiteDBAccess.java:196)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getDBAccess(LocalDynamoDBRequestHandler.java:198)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.getHandler(LocalDynamoDBRequestHandler.java:189)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBRequestHandler.createTable(LocalDynamoDBRequestHandler.java:232)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:18)
	at com.amazonaws.services.dynamodbv2.local.dispatchers.CreateTableDispatcher.enact(CreateTableDispatcher.java:12)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:364)
	at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:451)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1340)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:205)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1242)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:220)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:503)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
	at java.lang.Thread.run(Thread.java:745)
[2019-01-12 17:21:02] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=6cb6b03bd3d02a644f61d46e3fe52655a62d10feca28f74d669f8d81d2f5aebe
header name: X-Amz-Date : value: 20190112T222102Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 600bff86-5a07-f3e6-065c-fed9e038d92c
header name: amz-sdk-retry : value: 10/7570/450
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:21:02] [qtp1851898486-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 2ae41773-35b1-4ea7-b35c-9307b7c8bc56
header name: Date : value: Sat, 12 Jan 2019 22:21:02 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 500
response body: {"__type":"com.amazonaws.dynamodb.v20120810#InternalFailure","message":"The request processing has failed because of an unknown error, exception or failure."}

[2019-01-12 17:21:02] [main] INFO  s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[11]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[2]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[3]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[4]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[5]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[6]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[7]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[8]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[9]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[10]: wasn't disposed before finalizing
[2019-01-12 17:21:03] [Finalizer] WARN  com.almworks.sqlite4java - [sqlite] DB[1]: wasn't disposed before finalizing
[2019-01-12 17:21:04] [main] INFO  o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:21:05] [main] INFO  o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService 'taskScheduler'
[2019-01-12 17:21:07] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
[2019-01-12 17:21:07] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
[2019-01-12 17:21:07] [main] INFO  s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
[2019-01-12 17:21:08] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_1
[2019-01-12 17:21:08] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_2
[2019-01-12 17:21:08] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_3
[2019-01-12 17:21:08] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:21:08] [main] INFO  o.a.tomcat.util.net.NioSelectorPool - Using a shared selector for servlet write/read
[2019-01-12 17:21:08] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 53570 (http) with context path ''
[2019-01-12 17:21:08] [main] INFO  o.m.s.d.integration.GetVersionStep - Started GetVersionStep in 65.04 seconds (JVM running for 68.997)
[2019-01-12 17:21:09] [http-nio-auto-1-exec-1] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
[2019-01-12 17:21:09] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
[2019-01-12 17:21:10] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Completed initialization in 117 ms
[2019-01-12 17:21:10] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:21:10] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:21:10] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@6f690a3f, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@4b7ae0a5, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@65c9a4c1, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@3b04e0d7, org.springframework.test.context.support.DirtiesContextTestExecutionListener@1c9b3d7a, org.springframework.test.context.transaction.TransactionalTestExecutionListener@788984, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@28bfdae6, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@3421957e, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@71940e6d, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@35e48d72, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@63da19c2, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@5b68eaa2]
[2019-01-12 17:21:11] [Thread-25] INFO  o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService 'taskScheduler'
[2019-01-12 17:21:11] [Thread-25] INFO  o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:21:11] [Thread-25] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:21:11] [Thread-25] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Shutting down embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false'
[2019-01-12 17:21:11] [Thread-3] WARN  o.s.c.c.s.e.MultipleJGitEnvironmentRepository - Failed to delete temporary directory on exit: java.nio.file.AccessDeniedException: C:\Users\Moorthy\AppData\Local\Temp\config-repo-7165538557564826831\.git\objects\pack\pack-ac4cc2b653f3e9d9e60018dd8fbaa6fa735e7123.idx
[2019-01-12 17:34:59] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:34:59] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:34:59] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@2f3a0f37, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@7ec8db0c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@4f1a5b93, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@22ee1ad7, org.springframework.test.context.support.DirtiesContextTestExecutionListener@5dd2ea0a, org.springframework.test.context.transaction.TransactionalTestExecutionListener@65fa4a07, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@782143e8, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@2a4bd699, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6003eb60, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@571e2f15, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@2552181d, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@51d9fd30]
[2019-01-12 17:35:02] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:35:02] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:35:08] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-1263316836088489708/Application.yml
[2019-01-12 17:35:08] [main] INFO  o.s.c.c.s.e.NativeEnvironmentRepository - Adding property source: file:/C:/Users/Moorthy/AppData/Local/Temp/config-repo-1263316836088489708/application.yml
[2019-01-12 17:35:08] [main] INFO  o.s.c.b.c.PropertySourceBootstrapConfiguration - Located property source: CompositePropertySource {name='configService', propertySources=[MapPropertySource {name='https://github.com/Romeh/config.git/Application.yml'}, MapPropertySource {name='https://github.com/Romeh/config.git/application.yml'}]}
[2019-01-12 17:35:09] [main] INFO  o.m.s.d.integration.GetVersionStep - The following profiles are active: INTEGRATION_TEST
[2019-01-12 17:35:10] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Bootstrapping Spring Data repositories in DEFAULT mode.
[2019-01-12 17:35:11] [main] INFO  o.s.d.r.c.RepositoryConfigurationDelegate - Finished Spring Data repository scanning in 142ms. Found 1 repository interfaces.
[2019-01-12 17:35:11] [main] INFO  o.s.cloud.context.scope.GenericScope - BeanFactory id=26ae97b7-cd7d-38cc-b55f-c490d254bb23
[2019-01-12 17:35:12] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.retry.annotation.RetryConfiguration' of type [org.springframework.retry.annotation.RetryConfiguration$$EnhancerBySpringCGLIB$$8814b2ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:35:12] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$92511aca] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:35:12] [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$ae6b1dc7] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[2019-01-12 17:35:13] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat initialized with port(s): 0 (http)
[2019-01-12 17:35:13] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Initializing ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:35:13] [main] INFO  o.a.catalina.core.StandardService - Starting service [Tomcat]
[2019-01-12 17:35:13] [main] INFO  o.a.catalina.core.StandardEngine - Starting Servlet Engine: Apache Tomcat/9.0.12
[2019-01-12 17:35:13] [main] INFO  o.a.c.core.AprLifecycleListener - The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [C:\Program Files\Java\jdk1.8.0_77\jre\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\Program Files (x86)\HP SimplePass 2011\x64;C:\Program Files (x86)\HP SimplePass 2011\;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\Program Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Shared\12.0\DLLShared\;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\Internet Explorer;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\Git\cmd;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLI\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Python3.6\Scripts\;C:\Python3.6\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\MongoDB\Server\3.0\bin;C:\apache-maven-3.2.2\bin;C:\Python27_;C:\Python27_\Scripts;C:\Program Files\Git\bin;C:\Program Files\Java\jdk1.8.0_77\bin;C:\Users\Moorthy\AppData\Local\Microsoft\WindowsApps;C:\workspace\gradle-4.1\bin;c:\spark\bin;C:\Program Files (x86)\GnuWin32\bin;C:\software install\apache-ant-1.10.5\bin;C:\software install\apache-ant-1.10.5\lib;C:\Program Files (x86)\7-Zip;C:\Users\Moorthy\AppData\Roaming\npm;.]
[2019-01-12 17:35:13] [main] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring embedded WebApplicationContext
[2019-01-12 17:35:13] [main] INFO  o.s.web.context.ContextLoader - Root WebApplicationContext: initialization completed in 4735 ms
[2019-01-12 17:35:14] [main] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Starting embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false', username='sa'
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'characterEncodingFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'webMvcMetricsFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'formContentFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'requestContextFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.FilterRegistrationBean - Mapping filter: 'httpTraceFilter' to: [/*]
[2019-01-12 17:35:16] [main] INFO  o.s.b.w.s.ServletRegistrationBean - Servlet dispatcherServlet mapped to [/]
[2019-01-12 17:35:16] [main] INFO  o.h.jpa.internal.util.LogHelper - HHH000204: Processing PersistenceUnitInfo [
	name: default
	...]
[2019-01-12 17:35:16] [main] INFO  org.hibernate.Version - HHH000412: Hibernate Core {5.3.7.Final}
[2019-01-12 17:35:16] [main] INFO  org.hibernate.cfg.Environment - HHH000206: hibernate.properties not found
[2019-01-12 17:35:16] [main] INFO  o.h.annotations.common.Version - HCANN000001: Hibernate Commons Annotations {5.0.4.Final}
[2019-01-12 17:35:17] [main] INFO  org.hibernate.dialect.Dialect - HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
[2019-01-12 17:35:19] [main] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Initialized JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:35:20] [main] WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration$JpaWebMvcConfiguration - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
[2019-01-12 17:35:21] [main] INFO  o.s.b.a.e.web.EndpointLinksResolver - Exposing 2 endpoint(s) beneath base path '/actuator'
[2019-01-12 17:35:22] [qtp1693523523-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:35:22] [qtp1693523523-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: CreateTable
[2019-01-12 17:35:23] [qtp1693523523-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=61c52ee1c45428f247404258d73d1ebd2af33b79600f4b62a1ec974c0fe13e81
header name: X-Amz-Date : value: 20190112T223522Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 798ff406-c5d6-5db0-dc8e-39fd82de3295
header name: amz-sdk-retry : value: 0/0/500
header name: Content-Length : value: 235
header name: X-Amz-Target : value: DynamoDB_20120810.CreateTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"ProvisionedThroughput":{"ReadCapacityUnits":50,"WriteCapacityUnits":50}}

[2019-01-12 17:35:23] [qtp1693523523-39] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 16623c6e-98d5-4a14-808e-cd92fbf2932e
header name: x-amz-crc32 : value: 252900770
header name: Date : value: Sat, 12 Jan 2019 22:35:22 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 200
response body: {"TableDescription":{"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"TableStatus":"ACTIVE","CreationDateTime":1547332523.297,"ProvisionedThroughput":{"LastIncreaseDateTime":0.000,"LastDecreaseDateTime":0.000,"NumberOfDecreasesToday":0,"ReadCapacityUnits":50,"WriteCapacityUnits":50},"TableSizeBytes":0,"ItemCount":0,"TableArn":"arn:aws:dynamodb:ddblocal:000000000000:table/Employee"}}

[2019-01-12 17:35:23] [qtp1693523523-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:35:23] [qtp1693523523-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: ListTables
[2019-01-12 17:35:23] [qtp1693523523-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=45cb2ab8805201b1058cf2b29247f23709ea56bfe91df7d5a9cdc3a98db314ea
header name: X-Amz-Date : value: 20190112T223523Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 1c9c1e63-b090-c4d1-329e-ef9ea247181f
header name: amz-sdk-retry : value: 0/0/500
header name: Content-Length : value: 2
header name: X-Amz-Target : value: DynamoDB_20120810.ListTables
header name: Content-Type : value: application/x-amz-json-1.0
request body: [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=45cb2ab8805201b1058cf2b29247f23709ea56bfe91df7d5a9cdc3a98db314ea
header name: X-Amz-Date : value: 20190112T223523Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: 1c9c1e63-b090-c4d1-329e-ef9ea247181f
header name: amz-sdk-retry : value: 0/0/500
header name: Content-Length : value: 2
header name: X-Amz-Target : value: DynamoDB_20120810.ListTables
header name: Content-Type : value: application/x-amz-json-1.0
request body: {}


[2019-01-12 17:35:23] [qtp1693523523-33] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 10ce139c-f7b0-44a6-bf50-d0f982dbaf35
header name: x-amz-crc32 : value: 127223554
header name: Date : value: Sat, 12 Jan 2019 22:35:23 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 200
response body: {"TableNames":["Employee"]}

[2019-01-12 17:35:23] [qtp1693523523-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - accessKey: AKIAJCHWAATYZZ7KUI5Q	eu-west-1
[2019-01-12 17:35:23] [qtp1693523523-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - target: DeleteTable
[2019-01-12 17:35:23] [qtp1693523523-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Request] header name: Authorization : value: AWS4-HMAC-SHA256 Credential=AKIAJCHWAATYZZ7KUI5Q/20190112/eu-west-1/dynamodb/aws4_request, SignedHeaders=amz-sdk-invocation-id;amz-sdk-retry;content-length;content-type;host;user-agent;x-amz-date;x-amz-target, Signature=ead89e7e164a3755af12b87b567ec85fe5db1bfb443cdddc86d6407aae39fe38
header name: X-Amz-Date : value: 20190112T223523Z
header name: User-Agent : value: aws-sdk-java/1.11.336 Windows_10/10.0 Java_HotSpot(TM)_64-Bit_Server_VM/25.77-b03 java/1.8.0_77
header name: Connection : value: keep-alive
header name: Host : value: localhost:9000
header name: amz-sdk-invocation-id : value: ddd35455-31ab-a6fe-07e1-050b1a6dd588
header name: amz-sdk-retry : value: 0/0/500
header name: Content-Length : value: 24
header name: X-Amz-Target : value: DynamoDB_20120810.DeleteTable
header name: Content-Type : value: application/x-amz-json-1.0
request body: {"TableName":"Employee"}

[2019-01-12 17:35:23] [qtp1693523523-38] INFO  c.a.s.d.l.s.LocalDynamoDBServerHandler - [Response] header name: x-amzn-RequestId : value: 0b642840-1dc0-43eb-bdca-224878c94115
header name: x-amz-crc32 : value: 252900770
header name: Date : value: Sat, 12 Jan 2019 22:35:23 GMT
header name: Content-Type : value: application/x-amz-json-1.0
status: 200
response body: {"TableDescription":{"AttributeDefinitions":[{"AttributeName":"EmployeeID","AttributeType":"S"}],"TableName":"Employee","KeySchema":[{"AttributeName":"EmployeeID","KeyType":"HASH"}],"TableStatus":"ACTIVE","CreationDateTime":1547332523.297,"ProvisionedThroughput":{"LastIncreaseDateTime":0.000,"LastDecreaseDateTime":0.000,"NumberOfDecreasesToday":0,"ReadCapacityUnits":50,"WriteCapacityUnits":50},"TableSizeBytes":0,"ItemCount":0,"TableArn":"arn:aws:dynamodb:ddblocal:000000000000:table/Employee"}}

[2019-01-12 17:35:23] [main] INFO  c.a.s.d.l.s.access.LocalDBClient - Shutting down
[2019-01-12 17:35:23] [main] INFO  c.a.s.d.l.s.access.LocalDBClient - Shutting down
[2019-01-12 17:35:24] [main] INFO  s.d.s.w.PropertySourcedRequestMappingHandlerMapping - Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[2019-01-12 17:35:24] [main] INFO  o.s.s.c.ThreadPoolTaskExecutor - Initializing ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:35:25] [main] INFO  o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService 'taskScheduler'
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Context refreshed
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper - Found 1 custom documentation plugin(s)
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.s.ApiListingReferenceScanner - Scanning for api listing references
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_1
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_2
[2019-01-12 17:35:26] [main] INFO  s.d.s.w.r.o.CachingOperationNameGenerator - Generating unique operation named: handleUsingGET_3
[2019-01-12 17:35:27] [main] INFO  o.a.coyote.http11.Http11NioProtocol - Starting ProtocolHandler ["http-nio-auto-1"]
[2019-01-12 17:35:27] [main] INFO  o.a.tomcat.util.net.NioSelectorPool - Using a shared selector for servlet write/read
[2019-01-12 17:35:27] [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 53743 (http) with context path ''
[2019-01-12 17:35:27] [main] INFO  o.m.s.d.integration.GetVersionStep - Started GetVersionStep in 27.211 seconds (JVM running for 30.864)
[2019-01-12 17:35:29] [http-nio-auto-1-exec-1] INFO  o.a.c.c.C.[Tomcat].[localhost].[/] - Initializing Spring DispatcherServlet 'dispatcherServlet'
[2019-01-12 17:35:29] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Initializing Servlet 'dispatcherServlet'
[2019-01-12 17:35:29] [http-nio-auto-1-exec-1] INFO  o.s.web.servlet.DispatcherServlet - Completed initialization in 138 ms
[2019-01-12 17:35:30] [main] INFO  o.s.t.c.s.AbstractContextLoader - Could not detect default resource locations for test class [org.moorthi.springboot.dynamodbspring.integration.CucumberRoot]: no resource found for suffixes {-context.xml, Context.groovy}.
[2019-01-12 17:35:30] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]
[2019-01-12 17:35:30] [main] INFO  o.s.b.t.c.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@7499b725, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@6c78008c, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@334ba8f9, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@69276bd8, org.springframework.test.context.support.DirtiesContextTestExecutionListener@50bec946, org.springframework.test.context.transaction.TransactionalTestExecutionListener@7ceedfbd, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@128cbcb9, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@40276c9b, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@66757920, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@6c6c5164, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@1b724124, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@60cab565]
[2019-01-12 17:35:32] [Thread-25] INFO  o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService 'taskScheduler'
[2019-01-12 17:35:32] [Thread-25] INFO  o.s.s.c.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
[2019-01-12 17:35:32] [Thread-25] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean - Closing JPA EntityManagerFactory for persistence unit 'default'
[2019-01-12 17:35:32] [Thread-25] INFO  o.s.j.d.e.EmbeddedDatabaseFactory - Shutting down embedded database: url='jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=false'
[2019-01-12 17:35:32] [Thread-3] WARN  o.s.c.c.s.e.MultipleJGitEnvironmentRepository - Failed to delete temporary directory on exit: java.nio.file.AccessDeniedException: C:\Users\Moorthy\AppData\Local\Temp\config-repo-1263316836088489708\.git\objects\pack\pack-ac4cc2b653f3e9d9e60018dd8fbaa6fa735e7123.idx
